ehrql/backends/emis.py

```diff
@@ -30,6 +34,22 @@ class EMISBackend(SQLBackend):
         ehrql.tables.beta.smoketest,
     ]
 
+    EMIS_ORG_HASH = os.environ.get("EMIS_ORGANISATION_HASH", "emis_organisation_hash")
+    HASHED_ORG_COLUMN = sqlalchemy.literal(
+        EMIS_ORG_HASH, type_=sqlalchemy.VARCHAR
+    ).label("hashed_organisation")
```

Comment:
`os.environ` is available as `self.config` in the Backend classes (and in the Query Engine classes too) so it's best to grab things from there e.g.
https://github.com/opensafely-core/ehrql/blob/896a357381d9ecc1aab57fa6a76d7d8ec8db3d7d/ehrql/backends/tpp.py#L502-L505